.changeset/clever-timers-whisper.md

```diff
@@ -0,0 +1,6 @@
+---
+"@khanacademy/wonder-blocks-typography": minor
+"@khanacademy/wonder-blocks-tokens": minor
```

Comment:
Should these be majors as some of the font-sizes are changing? Or just minor?

Reply:
IMO it should be `minor` if only the values change. If the `font` structure changes (public API), then that could be considered as a breaking change.

TBH this is the kind of situation were any option could work :)

Reply:
Alright, I'll leave it as-is!

Reply:
Since we are using `sizing` tokens now for the `font.size` and `font.lineHeight` tokens, the token values will be strings now including the `rem` unit, instead of the numeric value. Would this be a breaking change since something like `fontSize: ${font.size.small}px` would need to be updated so it doesn't become `font-size: 1.4rempx`?

(Or, depending on if the css variable changes are released first, it'd become `font-size: var(--wb-token-name)px`)

Reply:
Great question. That does seem breaking to me as it will have to be updated at the call-sites to prevent an issue. I see numerous instances of `px` added after `font.size` and `font.lineHeight` tokens in webapp!